pkg/logs/client/http/destination.go

```diff
@@ -322,6 +324,7 @@ func (d *Destination) unconditionalSend(payload *message.Payload) (err error) {
 
 	defer resp.Body.Close()
 	response, err := io.ReadAll(resp.Body)
+	log.Tracef("Log agent payload sent with: %s", resp.Proto)
```

Comment:
I think you can remove this log since it is mostly a dupe of the above log line.

Reply:
Oh I see you are using `req.Proto` above. 
I think the above log line should be changed to `resp.Proto` as right now, it may not be accurate. + We only really care what the protocol ended up actually being after the request was made.

Reply:
Ah, actually i had a closer look at it. Seems like by default go with always send request as 1.1 but will eventually auto negotiate up to http2 if available. so logging both req and res might end up just showing mismatch (which is expected) between req.proto and resp.proto 

In that case, maybe we can just keep the response instead and use the req as logging for latency?

Reply:
Sorry I responded in another comment. But I agree, lets keep the response proto - but merge it all into one log line for simplicity.